### PR TITLE
Added Dep installation instructions

### LINKED ISSUE #164: Dep Support
Description
===========

I am wondering if there is already work being done to make this package available to be installed via dep? Glide works wonderfully but as [dep](https://github.com/golang/dep) is preferred now I am getting the output below. If there is no one working on this I am more than glad to. Thanks for this _awesome_ package by-the-way!

How to reproduce
================

```
$ dep ensure -add github.com/confluentinc/confluent-kafka-go

Fetching sources...

Solving failure: No versions of github.com/confluentinc/confluent-kafka-go met constraints:
	v0.11.0: Could not introduce github.com/confluentinc/confluent-kafka-go@v0.11.0, as its subpackage github.com/confluentinc/confluent-kafka-go does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	v0.9.4: Could not introduce github.com/confluentinc/confluent-kafka-go@v0.9.4, as its subpackage github.com/confluentinc/confluent-kafka-go does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	v0.9.2: Could not introduce github.com/confluentinc/confluent-kafka-go@v0.9.2, as its subpackage github.com/confluentinc/confluent-kafka-go does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	v0.9.4-RC2: Could not introduce github.com/confluentinc/confluent-kafka-go@v0.9.4-RC2, as its subpackage github.com/confluentinc/confluent-kafka-go does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	v0.9.4-RC1: Could not introduce github.com/confluentinc/confluent-kafka-go@v0.9.4-RC1, as its subpackage github.com/confluentinc/confluent-kafka-go does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	v0.9.2-RC1: Could not introduce github.com/confluentinc/confluent-kafka-go@v0.9.2-RC1, as its subpackage github.com/confluentinc/confluent-kafka-go does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	master: Could not introduce github.com/confluentinc/confluent-kafka-go@master, as its subpackage github.com/confluentinc/confluent-kafka-go does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	0.11.0.x: Could not introduce github.com/confluentinc/confluent-kafka-go@0.11.0.x, as its subpackage github.com/confluentinc/confluent-kafka-go does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	0.9.2.x: Could not introduce github.com/confluentinc/confluent-kafka-go@0.9.2.x, as its subpackage github.com/confluentinc/confluent-kafka-go does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	0.9.4.x: Could not introduce github.com/confluentinc/confluent-kafka-go@0.9.4.x, as its subpackage github.com/confluentinc/confluent-kafka-go does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	api_catchup: Could not introduce github.com/confluentinc/confluent-kafka-go@api_catchup, as its subpackage github.com/confluentinc/confluent-kafka-go does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	citest: Could not introduce github.com/confluentinc/confluent-kafka-go@citest, as its subpackage github.com/confluentinc/confluent-kafka-go does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	generrs094: Could not introduce github.com/confluentinc/confluent-kafka-go@generrs094, as its subpackage github.com/confluentinc/confluent-kafka-go does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	go1.8rc3: Could not introduce github.com/confluentinc/confluent-kafka-go@go1.8rc3, as its subpackage github.com/confluentinc/confluent-kafka-go does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	headers: Could not introduce github.com/confluentinc/confluent-kafka-go@headers, as its subpackage github.com/confluentinc/confluent-kafka-go does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	initial_api: Could not introduce github.com/confluentinc/confluent-kafka-go@initial_api, as its subpackage github.com/confluentinc/confluent-kafka-go does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	no_static_on_osx: Could not introduce github.com/confluentinc/confluent-kafka-go@no_static_on_osx, as its subpackage github.com/confluentinc/confluent-kafka-go does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	producer_timestamp: Could not introduce github.com/confluentinc/confluent-kafka-go@producer_timestamp, as its subpackage github.com/confluentinc/confluent-kafka-go does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	short_primer: Could not introduce github.com/confluentinc/confluent-kafka-go@short_primer, as its subpackage github.com/confluentinc/confluent-kafka-go does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	static_builds: Could not introduce github.com/confluentinc/confluent-kafka-go@static_builds, as its subpackage github.com/confluentinc/confluent-kafka-go does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	travis: Could not introduce github.com/confluentinc/confluent-kafka-go@travis, as its subpackage github.com/confluentinc/confluent-kafka-go does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	travis_lint: Could not introduce github.com/confluentinc/confluent-kafka-go@travis_lint, as its subpackage github.com/confluentinc/confluent-kafka-go does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	vercheck: Could not introduce github.com/confluentinc/confluent-kafka-go@vercheck, as its subpackage github.com/confluentinc/confluent-kafka-go does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)

```

Checklist
=========
Please provide the following information:

 - [x] confluent-kafka-go and librdkafka version (`LibraryVersion()`): 721919 & 0.11.3
 - [x] Apache Kafka broker version: N/A
 - [x] Client configuration: `ConfigMap{...}`: N/A
 - [x] Operating system: Mac/OSX 10.12.6
 - [x] Provide client logs (with `"debug": ".."` as necessary): Provided in reproduce
 - [x] Provide broker log excerpts: Not needed since dep issue I hope
 - [ ] Critical issue

### PR DESCRIPTION
**Pull request by lemoney**
fixes #164 

I added a blurb to the readme so that users installing via dep have an example command

### CODE REVIEW

**Comment by lemoney** [comment]
[clabot:check]

**Comment by lemoney** [comment]
[clabot:check]

**Comment by edenhill** [comment]
@lemoney Can you try to sign the CLA again so we can get this merged?
Thanks

**Comment by lemoney** [comment]
[clabot:check]

**Comment by edenhill** [comment]
@lemoney It seems like there's a problem signing the CLA, do you see any error messages when trying to sign it?

**Comment by edenhill** [comment]
[clabot:check]

File: README.md [hunk 1]
```diff
@@ -144,10 +144,16 @@ Build from source:
...
 -------------------
 
+installing via `go get`:
 ```
 go get -u github.com/confluentinc/confluent-kafka-go/kafka
 ```
 
+installing via `dep`:
+```
+dep ensure -add github.com/confluentinc/confluent-kafka-go/kafka
```
Comment (oivoodoo) [inline README.md]: Hey @lemoney . Are you sure about this change? I mean people are using all the time the different ways on handle the vendor folder. Also you can use `go get` and then when you started to use the package call `dep ensure` instead.

**Pull request closed without merging**